apple/Sources/FerrostarSwiftUI/Views/Controls/FerrostarButton.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+public struct FerrostarButtonStyle: ButtonStyle {
+    /// The ferrostar button style.
+    public init() {}
+
+    public func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .padding()
+            .background(Color(.systemBackground))
+            .foregroundStyle(.primary)
+            .clipShape(Capsule())
+            .frame(minWidth: 52, minHeight: 52)
+    }
+}
+
+public struct FerrostarButton<Label: View>: View {
+    let action: () -> Void
+    @ViewBuilder var label: () -> Label
+
+    /// The basic Ferrostar SwiftUI button style.
+    ///
+    /// - Parameters:
+    ///   - action: The action the button performs on tap.
+    ///   - label: The label subview.
+    public init(action: @escaping () -> Void, label: @escaping () -> Label) {
```

Comment:
Carrying on the same theme from the other discussion, I think this should be non-escaping and maybe a ViewBuilder? I wish I could point to the profound reasoning why I am convinced of this, but I'm pretty sure actions should be escaping but not views.